src/decorators/dispatch.spec.ts

```diff
@@ -108,7 +108,7 @@ describe('@dispatch', () => {
       ).toHaveBeenCalledWith(expectedArgs);
     });
 
-    it("shouldn't call dispatch", () => {
```

Comment:
**NOTE** there seems to be a new hook that runs before you commit. I had a single quoted string like `'shouldn\'t call dispatch.'` and the hook was silently changing my single quotes to double quotes, making the ci tool that runs here on GitHub fail the tests. I got rid of the escaped single quote inside the string and then it all worked: beware of this behaviour!